app/build.gradle.kts

```diff
@@ -418,6 +418,8 @@ dependencies {
 
   testImplementation(libs.truth)
 
+  testImplementation(libs.kotlin.reflect)
```

Comment:
Why do we need this dependency? 🤔

Reply:
The Quarantine library earlier was pulling this in transitively. Now that the library is removed, some tests started failing so I needed to add the dependency again.